src/FakeItEasy/Fake.cs

```diff
@@ -75,7 +75,7 @@ public static void ClearRecordedCalls(object fakedObject)
         /// <param name="fakeManager">The fake manager, or <c>null</c> if <paramref name="potentialFake"/> is not actually a faked object.</param>
         /// <returns><c>true</c> if <paramref name="potentialFake"/> is a faked object, else <c>false</c>.</returns>
         [DebuggerStepThrough]
-        public static bool TryGetFakeManager(object potentialFake, out FakeManager fakeManager)
+        public static bool TryGetFakeManager(object potentialFake, [NotNullWhen(true)]out FakeManager? fakeManager)

```

Comment:
I notice you didn't add a space after the attribute. In at least one other place, you did. I think it would be better with a space. If you don't want to change it now, we can do it in a subsequent PR.

Reply:
Oh. I wasn't thinking about it one way or another. Happy to add. Amend suits?

Reply:
> Amend okay?

Perfect